.github/workflows/release.yml

```diff
@@ -0,0 +1,86 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        description: "Prerelease"
+        required: true
+        default: false
+        type: boolean
+      draft:
+        description: "Draft"
+        required: true
+        default: false
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    name: Version Bump and Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AUTOMATION_USER_TOKEN }}
+          fetch-depth: 0
+
+      - name: Get latest non-prerelease release
+        id: latestrelease
+        uses: cardinalby/git-get-release-action@v1
+        with:
+          latest: true
+          prerelease: false
+          draft: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get next version
+        id: semverbump
+        uses: WyriHaximus/github-action-next-semvers@v1.2.1
+        with:
+          version: ${{ steps.latestrelease.outputs.tag_name }}
+
+      - name: Update Version in code
+        run: |
+          sed -i "s/\$this->container\['sdk_version'\] = '[0-9]\+\.[0-9]\+\.[0-9]\+';/\$this->container['sdk_version'] = '${{steps.semverbump.outputs.minor}}';/g" ./lib/Model/UserData.php
```

Comment:
PHP must have some basic "read in a file as text" function, right? This looks a little more brittle compared to the other SDKs since it's not replacing just a constant declaration or a file containing just the version string.

Reply:
probably.  I can look into that as an alternative option. I'm not PHP savvy so this seemed like the quickest way to success.

Reply:
I don't think it's essential at all, just vaguely worried that if these sed commands don't match something, they're just going to fail silently.

Reply:
ok, i push a version that has a local file which can be managed by the release action and is loaded when creating UserData objects.  I ran the example app and it doesn't seem to explode